app/controllers/item.js

```diff
@@ -78,6 +82,17 @@ export default Ember.Controller.extend({
     }
   }),
 
+  setAndRedirectToCategory(category) {
+    const parentId = category.get("parentId");
+    this.transitionToRoute(
+      "package_category",
+      parentId ? parentId : category.id
+    );
+    if (parentId) {
```

Comment:
I guess this should have been written before transition, any reason to write it below ?

Reply:
no reason, works bothways.